pkg/api/types.go

```diff
@@ -1889,6 +1889,8 @@ type EndpointAddress struct {
 	// Optional: Hostname of this endpoint
 	// Meant to be used by DNS servers etc.
 	Hostname string `json:"hostname,omitempty"`
+	// Optional: Node hosting this endpoint
```

Comment:
Can you comment on why this is here and what happens if it is not?

e.g. "This can be used to inform routing and load-balancing of service IP"